pkg/client/client.go

```diff
@@ -332,7 +333,16 @@ func (o *OptimizelyClient) Track(eventKey string, userContext entities.UserConte
 	}
 
 	userEvent := event.CreateConversionUserEvent(projectConfig, configEvent, userContext, eventTags)
-	o.EventProcessor.ProcessEvent(userEvent)
+	if o.EventProcessor.ProcessEvent(userEvent) && o.NotificationCenter != nil {
+		trackNotification := notification.TrackNotification{Type: notification.Track, EventKey: eventKey, UserContext: userContext, EventTags: eventTags}
+		var payload []interface{}
+		payload = append(payload, trackNotification, userEvent)
```

Comment:
Why make this an array? We can add the UserEvent to the track notification struct